components/tag-list/tag-list.js

```diff
@@ -354,8 +354,13 @@ class TagList extends LocalizeCoreElement(InteractiveMixin(ArrowKeysMixin(LitEle
 		setTimeout(() => focusableElem.focus(), this.clearFocusTimeout);
 	}
 
-	_handleKeyboardTooltipShown() {
-		this._hasShownKeyboardTooltip = true;
+	_handleKeyboardTooltipShown(e) {
+		if (!this._hasShownKeyboardTooltip) this._hasShownKeyboardTooltip = true;
+		else if (this.clearable && !this.contains(e.detail?.relatedTarget)) {
```

Comment:
What is the `relatedTarget` meant to indicate?

Reply:
Related target is the blurred element from the first `tag-list-item` focus event. So this is meant to run only if the focus is coming from outside the `tag-list`, preventing the announcement when navigating to the first item using arrow keys.